epub4j-core/src/test/java/io/documentnode/epub4j/epub/ResourcesLoaderTest.java

```diff
@@ -14,175 +14,176 @@
-  }
-
-  /**
-   * Loads all Resources lazily from a ZipFile
-   */
-  @Test
-  public void testLoadResources_ZipFile_lazy_all() throws IOException {
-    // given
-    ZipFile zipFile = new ZipFile(testBookFilename);
-
-    // when
-    Resources resources = ResourcesLoader.loadResources(zipFile, encoding,
-        Arrays.asList(MediaTypes.mediaTypes));
-
-    // then
-    verifyResources(resources);
-    Assert.assertEquals(Resource.class,
-        resources.getById("container").getClass());
-    Assert.assertEquals(LazyResource.class,
-        resources.getById("book1").getClass());
-  }
-
-  /**
-   * Loads the Resources from a ZipFile, some of them lazily.
-   */
-  @Test
-  public void testLoadResources_ZipFile_partial_lazy() throws IOException {
-    // given
-    ZipFile zipFile = new ZipFile(testBookFilename);
-
-    // when
-    Resources resources = ResourcesLoader.loadResources(zipFile, encoding,
-        Collections.singletonList(MediaTypes.CSS));
-
-    // then
-    verifyResources(resources);
-    Assert.assertEquals(Resource.class,
-        resources.getById("container").getClass());
-    Assert.assertEquals(LazyResource.class,
-        resources.getById("book1").getClass());
-    Assert
-        .assertEquals(Resource.class, resources.getById("chapter1").getClass());
-  }
-
-  private void verifyResources(Resources resources) throws IOException {
-    Assert.assertNotNull(resources);
-    Assert.assertEquals(12, resources.getAll().size());
-    List<String> allHrefs = new ArrayList<>(resources.getAllHrefs());
-    Collections.sort(allHrefs);
-
-    Resource resource;
-    byte[] expectedData;
-
-    // container
-    resource = resources.getByHref(allHrefs.get(0));
-    Assert.assertEquals("container", resource.getId());
-    Assert.assertEquals("META-INF/container.xml", resource.getHref());
-    Assert.assertNull(resource.getMediaType());
-    Assert.assertEquals(230, resource.getData().length);
-
-    // book1.css
-    resource = resources.getByHref(allHrefs.get(1));
-    Assert.assertEquals("book1", resource.getId());
-    Assert.assertEquals("OEBPS/book1.css", resource.getHref());
-    Assert.assertEquals(MediaTypes.CSS, resource.getMediaType());
-    Assert.assertEquals(74, resource.getData().length);
-    expectedData = IOUtil
-        .toByteArray(this.getClass().getResourceAsStream("/book1/book1.css"));
-    Assert.assertArrayEquals(expectedData, resource.getData());
-
-    // chapter1
-    resource = resources.getByHref(allHrefs.get(2));
-    Assert.assertEquals("chapter1", resource.getId());
-    Assert.assertEquals("OEBPS/chapter1.html", resource.getHref());
-    Assert.assertEquals(MediaTypes.XHTML, resource.getMediaType());
-    Assert.assertEquals(5100, resource.getData().length);
-    expectedData = IOUtil.toByteArray(
-        this.getClass().getResourceAsStream("/book1/chapter1.html"));
-    Assert.assertArrayEquals(expectedData, resource.getData());
-  }
+//  private static final String encoding = "UTF-8";
```

Comment:
Shall we have tests still?

Reply:
It's the same error on my computer when using unchangeable code.